packages/components/src/alignment-matrix-control/types.ts

```diff
@@ -10,6 +10,11 @@ export type AlignmentMatrixControlValue =
 	| 'bottom center'
 	| 'bottom right';
 
+export type VoidableAlignmentMatrixControlValue =
```

Comment:
I generally find those `Voidable*` types a bit unnecessary. It may be more readable to just inline `AlignmentMatrixControlValue | null | undefined`. 

Then we should ask ourselves in every one of those instances: which one of them really needs to support both `null` and `undefined` values? Can we minimize those scenarios?